output/csv/output.go

```diff
@@ -87,6 +89,7 @@ func newOutput(params output.Params) (*Output, error) {
 
 	saveInterval := config.SaveInterval.TimeDuration()
 	fname := config.FileName.String
+	timeFormat := config.TimeFormat
```

Comment:
```suggestion
	if config.TimeFormat != Unix && config.TimeFormat != RFC3339 {
	    return fmt.Errorf("the timeFormat provided is not supported")
	}
	
```

Reply:
Won't this continue to grow into a long and unmaintainable `if` statement as we add more time formats?  

This was the reason I made an `isValid()` method which isolates the validity test from the parser itself, and keeps the validation and type definitions within the same file `output/csv/consts.go`.  It's also easier to unit test as-is.

Reply:
Won't this continue to grow into a long and unmaintainable if statement as we add more time formats?

This was the reason I made an isValid() method which isolates the validity test from the parser itself, and keeps the validation and type definitions within the same file output/csv/consts.go. It's also easier to unit test as-is.